### ISSUE TITLE
Planning : Overlapping tasks

### ISSUE THREAD

**Issue by Richard76000** [glpi-project/glpi#8776]
Hello,

We have a problem with GLPI. We have implemented the scheduled task system for technicians, however when the same technician schedules the task for a day and a time when there is already another task with the same time on the same day, it just marks him a message of 'Warning:

"User ******* is busy for the selected time period
- Task ticket for one: from 02-17-2021 10:30 am to 02-17-2021 11:00 am
New Agent Form
- External event: from 02-17-2021 10:00 to 02-17-2021 12:00
busy "

We want this to block the user and not save the task.

Looking forward to your feedback,

Thank you very much.

Regards,

**Comment by cedric-anne** [comment]
Hi,

This feature does not exists in GLPI core. It may exists in a plugin but I never heard about this.

You may consider to discuss about this new feature with the community on our suggest website: https://suggest.glpi-project.org .

If you want to fund this feature, please don't hesitate to contact us: https://portal.glpi-network.com/contact-us

Regards

**Comment by Richard76000** [comment]
This is a bug, it must be possible to be able to prohibit an overlap of tasks. There it is just attention there is another task. The technician is totally lost afterwards in his schedule

**Comment by cedric-anne** [comment]
Even if it may be considered as an important feature for some GLPI usages, this is not a bug.

This warning has been introduced in 9.5 and we choosed to not block task creation. It is up to task creator to find a solution for this.
There can be many different situations and we cannot predict the choices that will be made.
 - Task may have an higher priority than the existing one, and so the other one may be re-scheduled in a second time.
 - Task creator may be constrained to create this task by some internal policy and have to warn a manager about this conflict.
 - Tasks may be planned on a loose schedule and not really requires 100% of technician capacity.
 - ...

I closed this issue as we only use Github issues for bugs.
